gnucash/gnome/gnc-plugin-page-register.cpp

```diff
@@ -4684,11 +4684,13 @@ gnc_plugin_page_register_cmd_goto_date (GSimpleAction *simple,
     Query* query;
     GList *splits;
     GtkWidget *window = gnc_plugin_page_get_window (GNC_PLUGIN_PAGE (page));
+    gboolean found = false;
```

Comment:
Don't mix types. Either (preferred) `bool found = false` or `gboolean found = FALSE`.

Reply:
Missed that, thanks.